evap/staff/tools.py

```diff
@@ -26,6 +27,24 @@ def delete_navbar_cache():
         cache.delete(key)
 
 
+def bulk_delete_users(request, username_file, test_run):
+    usernames = [u.strip() for u in username_file.readlines()]
+    users = UserProfile.objects.filter(username__in=usernames)
+    deletable_users = [u for u in users if u.can_staff_delete]
+
+    messages.info(request, 'The text file contains {} usernames. {} Users have been found in the database. {} of those are deletable.'
```

Comment:
Please use `The uploaded text file contains {} usernames. {} users have been found in the database. {} of those can be deleted.`